sx-question-list.el

```diff
@@ -592,6 +618,19 @@ Sets `sx-question-list--site' and then call
     (setq sx-question-list--site site)
     (sx-question-list-refresh 'redisplay)))
 
+(defun sx-question-list-order-by (sort)
+  "Order questions in the current list by the method SORT.
+Sets `sx-question-list--order' and then calls
+`sx-question-list-refresh' with `redisplay'."
+  (interactive
+   (list (when sx-question-list--order
```

Comment:
So the user won't be unnecessarily prompted if he's about to be given an error below.